modules/library/coverage/src/main/java/org/geotools/coverage/processing/operation/Crop.java

```diff
@@ -252,16 +256,42 @@
 			null,  // Maximal value
 			null, // Unit of measure
 			true); // Parameter is optional
+	
+        /**
+         * The parameter descriptor used to tell this operation to check NoData
+         */
+        public static final ParameterDescriptor<Range> NODATA = new DefaultParameterDescriptor<Range>(
+                        Citations.JAI, PARAMNAME_NODATA,
+                        Range.class, // Value class
+                        null, // Array of valid values
+                        null,  // Default value
+                        null,  // Minimal value
+                        null,  // Maximal value
+                        null, // Unit of measure
+                        true); // Parameter is optional
+        
+        /**
+         * The parameter descriptor used to tell this operation to set destinationNoData
+         */
+        public static final ParameterDescriptor<double[]> DEST_NODATA = new DefaultParameterDescriptor<double[]>(
```

Comment:
Interesting. A Crop is a pure geometric operation, cutting the image on a shape, why would one want to remap the nodata values in the process?